concepts/05 UI Components/Tooltip/02 Getting Started with Tooltip/00 Getting Started with Tooltip.md

```diff
@@ -0,0 +1,18 @@
+#include tutorials-intro-installationnote
+
+Tooltip is a UI component that allows you to display information over a specific element in your application.
+
+This tutorial guides you through the following steps: 
+
+- Add Tooltip to a page and configure core component settings.
+- Customize the appearance of Tooltip visual elements.
```

Comment:
I wouldn't say that a regular tooltip contains "elements". If it's just a popup with text in it, I'd just say it's one element. You can probably change font settings, background color, border style... and that's it. 

```suggestion
- Customize Tooltip appearance.
```